# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/fake/spicctrl/install-prefix.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2003-07-11 18:08:37.000000000 +0200
+++ ./Makefile	2003-07-11 18:08:47.000000000 +0200
@@ -6,7 +6,7 @@
 LIBS =
 DOCS = AUTHORS CHANGES LICENSE
 VERSION = 1.4
-PREFIX = /usr/local
+PREFIX = /usr
 DESTBIN = $(PREFIX)/bin
 DESTMAN = $(PREFIX)/man/man1
 DESTDOC = $(PREFIX)/share/doc/spicctrl-$(VERSION)
